EIPS/eip-4844.md

```diff
@@ -173,7 +173,7 @@ class BlobTransaction(Container):
     chain_id: uint256
     nonce: uint64
     priority_fee_per_gas: uint256
-    max_basefee_per_gas: uint256
```

Comment:
LGTM, to make it consistent with 1559.

Consider also change Line 175 and else where for `priority_fee_per_gas` -> `max_priority_fee_per_gas` to match with 1559?